fake_service/schema/tasks.py

```diff
@@ -1,17 +1,25 @@
 from celery import shared_task
 from django.core.cache import cache
-
 from schema.services import FakeDataWrite
+from schema.utils import generate_cache_task_key
 
 
 @shared_task()
-def download_dataset_task(cache_task_key: str, schema_id: str, count: str, user_id: str):
+def download_dataset_task(cache_task_key: str, schema_id: str, row_nums: str, user_id: str):
     try:
         cache.set(cache_task_key, {"is_importing": False, "errors": []}, timeout=300)
-        FakeDataWrite().write_data_to_db_and_remove_from_disk(schema_id=schema_id, count=count, user_id=user_id)
+        FakeDataWrite().write_data_to_db_and_remove_from_disk(schema_id=schema_id, row_nums=row_nums, user_id=user_id)
```

Comment:
also bad naming here. You have `download_dataset_task`  and call `write_data_to_db_and_remove_from_disk` inside